python/unityagents/environment.py

```diff
@@ -115,14 +115,14 @@ def __init__(self, file_name, worker_id=0,
                     .format(str(file_name)))
 
             if "apiNumber" not in p:
-                self._unity_api = "API-1"
+                self._unity_version = "API-1"
```

Comment:
Not a fan of this, might be easily confused with Unity Version, as in 2018.1, etc.

Reply:
I agree this could be problematic. Any suggestion for a better name?

Reply:
mlagents_python_version and mlagents_unity_version (?)

Reply:
Actually, why do we need two local vars .... we just need to create a const and check that it is different from p["apiNumber"] (?)

Reply:
We didn't have API version in v0.1, so we need to be able to catch Unity not providing one.